README.md

```diff
@@ -0,0 +1,59 @@
+# zabbix-aruba-sensor-monitoring
+
+This python module provides Zabbix monitoring support for Aruba sensors using the Aruba User Experience Insight API. This API is currently in closed beta state and subjective to change at any time.
+
+
+
+## Requirements
+
+- Zabbix agent
+- pip
+- requests (installed automatically as dependency)
+
+
+
+## Installation
+
+1. Install the python module using pip.
+
+```
+pip install https://github.com/digiaiiris/zabbix-aruba-sensor-monitoring/releases/download/1.0.0/aruba-sensor-monitoring-1.0.0.tar.gz
+```
+
+2. Copy the [Zabbix agent configuration](etc/zabbix/zabbix_agent.d/ic_aruba.conf) to /etc/zabbix/zabbix_agent.d directory.
+
+3. Restart the Zabbix agent.
+
+
+
+## Usage
+
+### Resource discovery
+
+Item Syntax | Description | Units |
+----------- | ----------- | ----- |
+aruba.discover.sensors[api_url, api_key, app_id] | Discover sensors from Aruba's status API. | {#SENSOR_NAME}, {#SENSOR_UID} |
```

Comment:
This is part of STIL-49 and not STIL-62 - I suggest you take it into the current sprint

Reply:
Updated STIL-49 to current sprint.